testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jaxrs/cfg/ResteasyAttributeTestCase.java

```diff
@@ -0,0 +1,365 @@
+        setAttributeValue(JaxrsAttribute.RESTEASY_PREFERJACKSONOVERJSONB, changeBoolean(JaxrsAttribute.RESTEASY_PREFERJACKSONOVERJSONB.getDefaultValue()));
+        setAttributeValue(JaxrsAttribute.RESTEASY_MEDIA_TYPE_PARAM_MAPPING, new ModelNode("abyz"));
+        setAttributeValue(JaxrsAttribute.RESTEASY_PROVIDERS, new ModelNode(Provider5.class.getName() + ", " + Provider6.class.getName() + " , " + StringTextStar.class.getName()));
+        setAttributeValue(JaxrsAttribute.RESTEASY_RESOURCES, new ModelNode(Resource1.class.getName() + "," + Resource2.class.getName()));
+        setAttributeValue(JaxrsAttribute.RESTEASY_RFC7232_PRECONDITIONS, changeBoolean(JaxrsAttribute.RESTEASY_RFC7232_PRECONDITIONS.getDefaultValue()));
+        setAttributeValue(JaxrsAttribute.RESTEASY_ROLE_BASED_SECURITY, changeBoolean(JaxrsAttribute.RESTEASY_ROLE_BASED_SECURITY.getDefaultValue()));
+        setAttributeValue(JaxrsAttribute.RESTEASY_SECURE_RANDOM_MAX_USE, changeInteger(JaxrsAttribute.RESTEASY_SECURE_RANDOM_MAX_USE.getDefaultValue()));
+        setAttributeValue(JaxrsAttribute.RESTEASY_USE_BUILTIN_PROVIDERS, changeBoolean(JaxrsAttribute.RESTEASY_USE_BUILTIN_PROVIDERS.getDefaultValue()));
+        setAttributeValue(JaxrsAttribute.RESTEASY_USE_CONTAINER_FORM_PARAMS, changeBoolean(JaxrsAttribute.RESTEASY_USE_CONTAINER_FORM_PARAMS.getDefaultValue()));
+        setAttributeValue(JaxrsAttribute.RESTEASY_WIDER_REQUEST_MATCHING, changeBoolean(JaxrsAttribute.RESTEASY_WIDER_REQUEST_MATCHING.getDefaultValue()));
+    }
+
+    /*
+     * Reset attributes to default values.
+     */
+    static void resetAttributeValues() throws IOException {
+        setAttributeValue(JaxrsAttribute.JAXRS_2_0_REQUEST_MATCHING, JaxrsAttribute.JAXRS_2_0_REQUEST_MATCHING.getDefaultValue());
+        setAttributeValue(JaxrsAttribute.RESTEASY_ADD_CHARSET, JaxrsAttribute.RESTEASY_ADD_CHARSET.getDefaultValue());
+        setAttributeValue(JaxrsAttribute.RESTEASY_BUFFER_EXCEPTION_ENTITY, JaxrsAttribute.RESTEASY_BUFFER_EXCEPTION_ENTITY.getDefaultValue());
+        setAttributeValue(JaxrsAttribute.RESTEASY_DISABLE_HTML_SANITIZER, JaxrsAttribute.RESTEASY_DISABLE_HTML_SANITIZER.getDefaultValue());
+        setAttributeValue(JaxrsAttribute.RESTEASY_DISABLE_PROVIDERS, new ModelNode(""));
+        setAttributeValue(JaxrsAttribute.RESTEASY_DOCUMENT_EXPAND_ENTITY_REFERENCES, JaxrsAttribute.RESTEASY_DOCUMENT_EXPAND_ENTITY_REFERENCES.getDefaultValue());
+        setAttributeValue(JaxrsAttribute.RESTEASY_DOCUMENT_SECURE_DISABLE_DTDS, JaxrsAttribute.RESTEASY_DOCUMENT_SECURE_DISABLE_DTDS.getDefaultValue());
+        setAttributeValue(JaxrsAttribute.RESTEASY_DOCUMENT_SECURE_PROCESSING_FEATURE, JaxrsAttribute.RESTEASY_DOCUMENT_SECURE_PROCESSING_FEATURE.getDefaultValue());
+        setAttributeValue(JaxrsAttribute.RESTEASY_GZIP_MAX_INPUT, JaxrsAttribute.RESTEASY_GZIP_MAX_INPUT.getDefaultValue());
+        setAttributeValue(JaxrsAttribute.RESTEASY_JNDI_RESOURCES, new ModelNode(""));
```

Comment:
Actually this is wrong and should be undefine-attribute

Reply:
Done